iOS Example Frame SPM/iOS Example Frame Regression Tests/CardValidationSnapshotTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CardValidationSnapshotTests.swift
+//  iOS Example Frame Regression Tests
+//
+//  Created by Okhan Okbay on 01/08/2023.
+//  Copyright © 2023 Checkout. All rights reserved.
+//
+
+import Frames
+import SnapshotTesting
+import XCTest
+
+final class CardValidationSnapshotTests: XCTestCase {
+    func testExpiryDateFormat_Success() {
+        let app = XCUIApplication()
+        app.launchFrames()
+
+        let expiryTextField = app.otherElements[AccessibilityIdentifiers.PaymentForm.cardExpiry]
+        app.enterText("1228", into: expiryTextField)
+        app.tapDoneButton()
+
+        assertSnapshot(matching: expiryTextField.screenshot().image, as: .image, record: false)
+    }
+
+    func testExpiryDateFormat_Failure() {
+        let app = XCUIApplication()
+        app.launchFrames()
+
+        let expiryTextField = app.otherElements[AccessibilityIdentifiers.PaymentForm.cardExpiry]
+        app.enterText("521", into: expiryTextField)
+        app.tapDoneButton()
+
+        assertSnapshot(matching: expiryTextField.screenshot().image, as: .image, record: false)
```

Comment:
Do we not get back the "Expiry Date in the past error" at the code level to add an automation test without using a snapshot ? . Just trying to understand what was the purpose to use a snapshot in this case looking at the screen shot below.

Reply:
We do get it, but it's only a verification from all the `staticTexts` available on that screen. 

The success snapshot in this PR tests the expiry date formatting `MM/YY` and the failure case tests if we're showing the right error and if the error is shown below the expiry date field.